docs/reference/configuration/keywords.rst

```diff
@@ -677,43 +677,145 @@ FIELD
 -----
 .. _field:
 
-The FIELD keyword is used to parametrize quantities which have extent over the
-full grid. In order to use the FIELD keyword, the GRID keyword must be supplied.
+The ``FIELD`` keyword is used to parametrize quantities that span the entire grid,
+with porosity and permeability being the most common examples.
+In order to use the ``FIELD`` keyword, the :ref:`GRID<grid>` keyword must be supplied.
 
-A parameter field (e.g. porosity or permeability or Gaussian Random Fields from APS) is defined as follows:
+Field parameters (e.g. porosity, permeability or Gaussian Random Fields from APS) are defined as follows:
 
 ::
 
-        FIELD  ID PARAMETER   <ECLIPSE_FILE>  INIT_FILES:/path/%d  MIN:X MAX:Y OUTPUT_TRANSFORM:FUNC INIT_TRANSFORM:FUNC  FORWARD_INIT:True
+        FIELD  ID  PARAMETER  <ECLIPSE_FILE>  INIT_FILES:/path/%d  FORWARD_INIT:True  INIT_TRANSFORM:FUNC  OUTPUT_TRANSFORM:FUNC  MIN:X  MAX:Y
 
-Here ID must be the same as the name of the parameter in the INIT_FILES.
-ECLIPSE_FILE is the name of the file ERT will export this field to when
-running simulations. Note that there should be an IMPORT statement in
-the ECLIPSE data file corresponding to the name given with ECLIPSE_FILE in case
-the field parameter is a field used in ECLIPSE data file like perm or poro.
-INIT_FILES is a filename (with an embedded %d if FORWARD_INIT is set to False)
-to load the initial field from. Can be RMS ROFF format, ECLIPSE restart format
-or ECLIPSE GRDECL format.
+- **ID**
+  Arbitrary string identifier that must match the name of the parameter specified in ``INIT_FILES``.
```

Comment:
There is a character limit of 8 characters here I believe, though not enforced in the config (yet), the code will crash if it is longer.

Reply:
Fixed, thanks.